src/patternfly/components/Expandable/docs/code.md

```diff
@@ -0,0 +1,18 @@
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-expanded="true"` | `.pf-c-expandable__toggle` | Indicates that the expandable content is visible. **Required** |
+| `aria-expanded="false"` | `.pf-c-expandable__toggle` | Indicates the the expandable content is hidden. **Required** |
+| `hidden` | `.pf-c-expandable__content` | Indicates that the expandable content is hidden. **Required**. |
```

Comment:
you might indicate that it's required when the menu is hidden, or maybe list this the way you did in the accordion.

> Indicates that the expandable content element is hidden. Use with `aria-expanded="false"` **Required** |